src/ectrans4py/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+if(HAVE_ETRANS)
```

Comment:
Because `HAVE_ETRANS` is a condition of the `ECTRANS4PY` feature, I think this if statement is not necessary.